README.md

```diff
@@ -1,84 +1,64 @@
-![Spiget Downloads](https://img.shields.io/spiget/downloads/100004?label=Spigot%20Downloads&color=%23ee8b1a)
-![Modrinth Downloads](https://img.shields.io/modrinth/dt/authy?label=Modrinth%20Downloads&color=%231bd96a)
+# <img align="left" width="100" height="100" src="https://github.com/Iru21/Authy/assets/12859907/010ce25c-4e45-43b9-bf3e-3e7e698adc52">Authy - a simple authentication plugin for Minecraft 1.17+!
 
-# Authy 
-A simple authentication plugin for Minecraft 1.17+!
+![Spigot Downloads](https://img.shields.io/spiget/downloads/100004?label=Spigot%20Downloads&color=%23ee8b1a) ![Modrinth Downloads](https://img.shields.io/modrinth/dt/authy?label=Modrinth%20Downloads&color=%231bd96a)
 
-Also get it on [Spigot](https://www.spigotmc.org/resources/authy.100004/) and [Modrinth](https://modrinth.com/plugin/authy)!
+_Click one of the badges above to get it!_
```

Comment:
Replace these badges with the ones from [here](https://intergrav.github.io/devins-badges-docs/badges/cozy/) and wrap them in links so it actually takes you to the site, and also looks better than the old ones